src/Components/ProductCard/ProductCard.css

```diff
@@ -1,8 +1,9 @@
 .ProductCardWrapper {
     min-width: 150px;
     height: 200px;
-    background-color: white;
-    border-radius: 10px;
+    background-color: #ffff;
+    border-radius: 3px;
+    border: 1px solid #ffff;
```

Comment:
HEX color code #ffffff

Reply:
yes, thanks! Will push the changes in the next PR.